examples/gnn_explainer_ba_shapes.py

```diff
@@ -83,12 +82,11 @@ def test():
         explanation = explainer(data.x, data.edge_index, index=node_index,
                                 target=data.y, edge_weight=data.edge_weight)
```

Comment:
The description in `Explainer` for the target is:
>   If the explanation type is :obj:`"phenomenon"`, the target has to be provided. If the explanation type is 
     :obj:`"model"`, the target should be set to :obj:`None` and will get automatically inferred.

However, we set the target in both cases here.

Reply:
Good point. Let me make a TODO to clarify this.